{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-14349",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2020-08/msg00044.html",
                    "name": "openSUSE-SU-2020:1228",
                    "refsource": "SUSE",
                    "tags": []
                },
                {
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2020-08/msg00049.html",
                    "name": "openSUSE-SU-2020:1244",
                    "refsource": "SUSE",
                    "tags": []
                },
                {
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2020-08/msg00050.html",
                    "name": "openSUSE-SU-2020:1243",
                    "refsource": "SUSE",
                    "tags": []
                },
                {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1865744",
                    "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1865744",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https://security.gentoo.org/glsa/202008-13",
                    "name": "GLSA-202008-13",
                    "refsource": "GENTOO",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "It was found that PostgreSQL versions before 12.4, before 11.9 and before 10.14 did not properly sanitize the search_path during logical replication. An authenticated attacker could use this flaw in an attack similar to CVE-2018-1058, in order to execute arbitrary SQL command in the context of the user used for replication."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-08-24T13:15Z",
    "lastModifiedDate": "2020-08-27T00:15Z"
}